proto/log_file.proto

```diff
@@ -10,15 +11,20 @@ import "proto/claim.proto";
     structured.
 */
 message LogFile {
+
+
+    fixed32 magic_number = 1;  // should be 1263421767
+
```

Comment:
Spacing and comment supposed to be there?

Reply:
Adding another comment to clarify.